src/formatDate.js

```diff
@@ -50,7 +50,41 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , separator] = fromFormat;
+  const [, , , newSeparator] = toFormat;
+
+  const dateArray = date.split(separator);
+  const newDateArray = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    if (fromFormat[i] === 'YY') {
+      if (+dateArray[i] < 30) {
+        dateArray[i] = dateArray[i].padStart(4, '20');
+      } else {
+        dateArray[i] = dateArray[i].padStart(4, '19');
+      }
```

Comment:
make sense to use ternary operator here as it will make a code shorter 
check other places

Reply:
code works with ternary operator, but linter does not let it to pass.
![image](https://github.com/mate-academy/js_format_date_advanced/assets/96629199/93b31245-4538-4639-ade4-50eb9861bc92)

Reply:
![image](https://github.com/mate-academy/js_format_date_advanced/assets/96629199/873d9f30-7e54-4bd2-a803-9688de071134)